### PR TITLE
Fix metric ProxySyncTimeTickLag

### LINKED ISSUE #23212: [Bug]: metric `ProxySyncTimeTickLag` only reports default ts
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

I think it's better to let ProxySyncTimeTickLag reports all channels' ts.

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Milvus Log

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by longjiquan**
fix: #23212 
/kind bug

### CODE REVIEW

File: internal/proxy/proxy.go [hunk 1]
```diff
@@ -310,8 +310,16 @@ func (node *Proxy) sendChannelsTimeTickLoop() {
 					Timestamps:       tss,
 					DefaultTimestamp: maxTs,
 				}
-				sub := tsoutil.SubByNow(maxTs)
-				metrics.ProxySyncTimeTickLag.WithLabelValues(strconv.FormatInt(paramtable.GetNodeID(), 10), "default").Set(float64(sub))
+
+				func() {
+					for channel, ts := range stats {
+						sub := tsoutil.SubByNow(ts)
```
Comment (jaime0815) [inline internal/proxy/proxy.go:316]: using maxTs should be enough, generally, we pay more attention to max lag.

Comment (longjiquan) [reply internal/proxy/proxy.go:316]: Normally, the default ts won't lag the latest time too much, so the metric turns to be useless.

Comment (longjiquan) [reply internal/proxy/proxy.go:316]: Btw, we should use the min ts to report the lag if we want to pay more attention to max lag.

**Pull request merged**